File: server/AppSupport.Data/Entities/Manifest.cs
using System;
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class Manifest
    {
        public int Id { get; set; }
        public int OrganizationId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateExpected { get; set; }
        public DateTime? DateExecuted { get; set; }
        public bool IsClosed { get; set; }

        public Organization Organization { get; set; }

        public IEnumerable<ManifestPlane> ManifestPlanes { get; set; }
    }
}

File: client/core/api/models/manifest.ts
import { ManifestPlane } from './manifest-plane';
import { Organization } from './organization';

export interface Manifest{
  id: number;
  organizationId: number;
  title: string;
  description: string;
  dateCreated: Date;
  dateExpected: Date;
  dateExecuted: Date;
  isClosed: boolean;

  organization: Organization;

  manifestPlanes: ManifestPlane[];
}
